crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -0,0 +1,2349 @@
+    const EInvalidUser: u64 = 12;
+    const ENotEqual: u64 = 13;
+    const EInvalidRestriction: u64 = 14;
+    const ELevelNotEmpty: u64 = 15;
+    const EInvalidPair: u64 = 16;
+    const EInvalidBaseBalance: u64 = 17;
+    const EInvalidFee: u64 = 18;
+    const EInvalidExpireTimestamp: u64 = 19;
+    const EInvalidTickSizeLotSize: u64 = 20;
+
+    // <<<<<<<<<<<<<<<<<<<<<<<< Error codes <<<<<<<<<<<<<<<<<<<<<<<<
+
+    // <<<<<<<<<<<<<<<<<<<<<<<< Constants <<<<<<<<<<<<<<<<<<<<<<<<
+    const FLOAT_SCALING: u64 = 1_000_000_000;
+    // Self-Trade Prevention option
+    // Cancel smaller order and decrement larger order by the smaller size. If the same size, cancel both.
+    const DECREMENT_AND_CANCEL: u8 = 0;
+    // Cancel older (resting) order in full. Continue to execute the newer taking order.
+    const CANCEL_OLDEST: u8 = 1;
+    // Cancel newer (taking) order in full. Let the old resting order remain on the order book.
+    const CANCEL_NEWEST: u8 = 2;
+    // Cancel both orders immediately.
+    const CANCEL_BOTH: u8 = 3;
+    // Restrictions on limit orders.
+    const N_RESTRICTIONS: u8 = 4;
+    const NO_RESTRICTION: u8 = 0;
+    // Mandates that whatever amount of an order that can be executed in the current transaction, be filled and then the rest of the order canceled.
+    const IMMEDIATE_OR_CANCEL: u8 = 1;
+    // Mandates that the entire order size be filled in the current transaction. Otherwise, the order is canceled.
+    const FILL_OR_KILL: u8 = 2;
+    // Mandates that the entire order be passive. Otherwise, cancel the order.
+    const POST_OR_ABORT: u8 = 3;
+    const MIN_BID_ORDER_ID: u64 = 1;
+    const MIN_ASK_ORDER_ID: u64 = 1 << 63;
+    const MIN_PRICE: u64 = 0;
+    const MAX_PRICE: u64 = ((1u128 << 64 - 1) as u64);
+    const TIMESTAMP_INF: u64 = ((1u128 << 64 - 1) as u64);
+    const REFERENCE_TAKER_FEE_RATE: u64 = 5_000_000;
+    const REFERENCE_MAKER_REBATE_RATE: u64 = 2_500_000;
+    const FEE_AMOUNT_FOR_CREATE_POOL: u64 = 100 * 1_000_000_000; // 100 SUI
+    const PREVENT_SELF_MATCHING_DEFAULT: u64 = 0;
+
+    // <<<<<<<<<<<<<<<<<<<<<<<< Constants <<<<<<<<<<<<<<<<<<<<<<<<
+
+    // <<<<<<<<<<<<<<<<<<<<<<<< Events <<<<<<<<<<<<<<<<<<<<<<<<
+
+    /// Emitted when a new pool is created
+    struct PoolCreated has copy, store, drop {
+        /// object ID of the newly created pool
+        pool_id: ID,
+        base_asset: TypeName,
+        quote_asset: TypeName,
+        taker_fee_rate: u64,
+        // 10^9 scaling
+        maker_rebate_rate: u64,
+        tick_size: u64,
+        lot_size: u64,
+    }
+
+    /// Emitted when a maker order is injected into the order book.
+    struct OrderPlaced<phantom BaseAsset, phantom QuoteAsset> has copy, store, drop {
+        /// object ID of the pool the order was placed on
+        pool_id: ID,
+        /// ID of the order within the pool
+        order_id: u64,
+        /// ID of client that placed the order
+        client_id: u64,
```

Comment:
Can we please call this `client_order_id` and update the comment? This is a client-defined identifier for the order, not an identifier of the client.

Reply:
sure!